doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1194,7 +1194,14 @@ Binding arguments of a constant to an interpretation scope
 Binding types of arguments to an interpretation scope
 +++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-.. cmd:: Bind Scope @scope with @qualid
+.. cmd:: Bind Scope @ident with {+ @class }
+
+   .. insertprodn class class
+
+   .. prodn::
+      class ::= Funclass
+      | Sortclass
+      | @smart_global
```

Comment:
Should rather be in the implicit coercion chapter.

Reply:
Done.

FWIW, I'm seeing messages like this appear incorrectly when I repeatedly rebuild the doc.  Probably some state information is not updated that should be.  A second run may fix things---still experimenting.

```
/mnt/c/proj/Coq2/doc/sphinx/addendum/implicit-coercions.rst:30: WARNING: Duplicate object: ('token', 'class'); other is at /mnt/c/proj/Coq2/doc/sphinx/user-extensions/syntax-extensions.rst
```

Reply:
The caching mechanism of the Sphinx build is clearly far from sound. I have another issue on my side, which is that it gets always invalidated now, and the build re-runs from scratch, making it quite long and annoying.

Reply:
It would be nice to have a "clean_sphinx" target in the makefile.